.github/pull_request_template.md

```diff
@@ -2,16 +2,28 @@
 
 ## Description
 
+<!-- Add a brief description of changes here -->
+
+## Screenshots (if appropriate):
+
+## How to Test
+
+<!-- Add steps to test this change. Include any other necessary relevant links -->
+
 <!--
 Replace PATH_TO_FEATURE with Vercel link
 -->
 
 :technologist: [Admin preview](PATH_TO_FEATURE)
 
-:rose: [Rose preview](PATH_TO_FEATURE)
-
 :desktop_computer: [Desktop preview](PATH_TO_FEATURE)
 
-<!-- Add a brief description of changes here. Include any other necessary relevant links -->
+## Checklist:
+<!-- strikethrough the checklist item that is not relevant to your change -->
 
-### Screenshots (if appropriate):
+- [ ] I have added before and after screenshots for UI changes
+- [ ] I have added JSON response output for API changes
+- [ ] I have added steps to reproduce and test for bug fixes in the description
+- [ ] I have commented on my code, particularly in hard-to-understand areas
+- [ ] My changes generate no new warnings
```

Comment:
I'm unsure about 'no new warnings'. Why would someone raise a PR introducing new warnings, even if it happens, why would PR be approved?

Reply:
I added this for this particular situation in mind - what if the contributor uses a deprecated method that they were part of previous ember/ember-data versions since this was a recent change 

Often times during the code review process, the approvers don't check the console for warnings. 

Happy to remove this if this seems unnecessary and won't be too helpful for the contributors.  Thoughts on this?

Reply:
Ty for the ember-data example. Happy to leave it as is and evaluate usage at a future date.